go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -469,3 +428,45 @@ func (c *addressManager) sync() {
 		}
 	}
 }
+
+// updateOVNEncapIP updates encap IP to OVS when the node primary IP changed.
+func updateOVNEncapIPAndReconnect(newIP net.IP) {
+	checkCmd := []string{
+		"get",
+		"Open_vSwitch",
+		".",
+		"external_ids:ovn-encap-ip",
+	}
+	encapIP, stderr, err := util.RunOVSVsctl(checkCmd...)
+	if err != nil {
+		klog.Warningf("Unable to retrieve configured ovn-encap-ip from OVS: %v, %q", err, stderr)
+	} else {
+		encapIP = strings.TrimSuffix(encapIP, "\n")
+		if len(encapIP) > 0 && newIP.String() == encapIP {
+			klog.V(4).Infof("Will not update encap IP, value: %s is the already configured", newIP)
```

Comment:
```suggestion
			klog.V(4).Infof("Will not update encap IP %s - it is already configured", newIP.String())
```
Not sure if my .String() is right but I expected something like this if youre using `%s` formatter.